setup.py

```diff
@@ -1,19 +1,46 @@
-import setuptools
+import os
+import sys
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
+from setuptools import setup
+from setuptools import find_packages
+from setuptools.command.install import install
 
-setuptools.setup(
+VERSION = "0.5.35"
+
+
+def readme():
+    with open("README.md", "r") as fh:
+        return fh.read()
+
+
+class VerifyVersionCommand(install):
```

Comment:
It might be cool to make this into a pre-push hook so we avoid it even getting to CI

Reply:
This specifically confirms that version that is checked out on CI matches the version that setup.py is trying to run.  Doing this in pre-hook wouldn't accomplish the same task.

Having both isn't a bad idea.  It would keep me from forgetting to upload a tag without the appropriate version defined in setup.py

Reply:
Exactly!